crates/bevy_text/src/text.rs

```diff
@@ -5,11 +5,20 @@ use bevy_render::color::Color;
 use bevy_utils::default;
 use serde::{Deserialize, Serialize};
 
+//use crate DEFAULT_FONT_HANDLE;
 use crate::Font;
-
+// TODO: reexport cosmic_text and these types in the prelude
```

Comment:
This todo should be fixed in this PR.

Reply:
Do we wish to re-export cosmic-text in the prelude of entire bevy?

Reply:
I think we should export these specific types in the prelude, and then also re-export cosmic-text, but *not* put it in the prelude.